pkgs/tools/misc/nb/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit }:
+
+let
+  version = "5.5.1";
+  name = "nb-${version}";
+in stdenv.mkDerivation {
+  inherit name;
+
+  src = fetchgit {
+    url    = "https://github.com/xwmx/nb.git";
+    rev    = version;
+    sha256 = "LXgLzO+rPCalkWBF79U2Y8r+wtTfSMMuHRYlT/wKd3g=";
+  };
```

Comment:
I believe currently `fetchFromGitHub` is preferred over `fetchgit if no submodules are needed?

Reply:
Interesting, where should I look to find conventions like this?
I don't have time to fix this up right now, but probably will this week (otherwise updates welcome).